projects/plugins/jetpack/_inc/client/lib/jp-redirect/index.js

```diff
@@ -25,6 +30,7 @@
  */
 export default function getRedirectUrl( source, args = {} ) {
 	const queryVars = {};
+	const calypsoEnv = get( window.Initial_State, 'calypsoEnv' );
```

Comment:
Maybe we could without lodash here, and instead rely on optional chaining?

Reply:
Yes! Please refer to p1HpG7-bRQ-p2